mscore/pagesettings.ui

```diff
@@ -247,6 +247,13 @@
      <property name="bottomMargin">
       <number>0</number>
      </property>
+     <item>
+      <widget class="QPushButton" name="resetPageStyleButton">
+       <property name="text">
+        <string>Reset All Page Sizes to Default</string>
```

Comment:
String should be: "Reset All Page *Settings* to Default"

Reply:
I think we need the help of @Tantacrul here. What text should be in the button for resetting the page styles (this button is located in Format -> Page Settings)?

Reply:
The button string should match the name of the dialog, like:

- Preferences dialog: "Reset All Preferences to Default"
- Style dialog: "Reset All Styles to Default"

Unless there are some things in the Page Settings dialog that are not affected by the reset?

Reply:
Changes text of this button, thanks